script.js

```diff
@@ -0,0 +1,689 @@
+          <span class="case off">Ь</span>
+          <span class="case on">ь</span>
+        </span>
+        <span class="digitM off">
+          <span class="case off">M</span>
+          <span class="case on">m</span>
+        </span>
+    </div>
+    <div class="key" id='Comma'>
+        <span class="digit on">
+          <span class="case off">Б</span>
+          <span class="case on">б</span>
+        </span>
+        <span class="digitB off">
+          <span class="case off"><</span>
+          <span class="case on"><</span>
+        </span>
+    </div>
+    <div class="key" id='Period'>
+        <span class="digitB on">
+          <span class="case off">Ю</span>
+          <span class="case on">ю</span>
+        </span>
+        <span class="digitB off">
+          <span class="case off">></span>
+          <span class="case on">></span>
+        </span>
+    </div>
+    <div class="key" id="Slash">
+        <span class="digit? on">
+          <span class="case off">,</span>
+          <span class="case on">.</span>
+        </span>
+        <span class="digit? off">
+          <span class="case off">?</span>
+          <span class="case on">/</span>
+        </span>
+    </div>
+    <div class="key" id="ArrowUp">
+      <span class="case arrow_up">▲</span>
+    </div>
+    <div class="key shift" id="ShiftRight">
+      <span class="case shift on">
+        <span class="case shift off">Shift</span>
+        <span class="case shift on">Shift</span>
+      </span>
+      <span class="case shift off">
+        <span class="case shift off">Shift</span>
+        <span class="case shift on">Shift</span>
+      </span>
+    </div>
+  </div>
+  <div class="keyboard_row">
+    <div class="key"id='ControlLeft'>
+      <span class="case ctrl">Ctrl</span>
+    </div>
+    <div class="key" id = 'OSLeft'>
+      <span class="case win">Win</span>
+    </div>
+    <div class="key alt" id="AltLeft">
+      <span class="case alt_left">Alt</span>
+    </div>
+    <div class="key space" id='Space'>
+      <span class="space on">
+        <span class="case on"> </span>
+      </span>
+    </div>
+    <div class="key alt" id='AltRight'>
+        <span class="case alt_right">Alt</span>
+    </div>
+    <div class="key" id='ArrowLeft'>
+        <span class="case arrow_left">◄</span>
+    </div>
+    <div class="key" id='ArrowDown'>
+        <span class="case arrow_down">▼</span>
+    </div>
+    <div class="key" id='ArrowRight'>
+      <span class="case arrow_right">►</span>
+    </div>
+    <div class="key" id="ControlRight">
+      <span class="case ctrl">Ctrl</span>
+    </div>
+  </div>
+    </div>       
+</div>`);
+
+let textarea = document.getElementById('name');
+let changeLang = document.querySelectorAll('.key>.on');
+let changeLang2 = document.querySelectorAll('.key>.off');
+
+document.onkeydown = function(e){
+  let capsButton = document.querySelectorAll(".on>.case");
+  let activeKey = document.getElementsByClassName("key");
```

Comment:
Use plural form for arrays (`capsButtons`, `activeKeys`), to say that your names are confusing is to say nothing